manifests/p/Poly/PlantronicsHub/3.25.54065.37203/Poly.PlantronicsHub.installer.yaml

```diff
@@ -1,20 +1,20 @@
-# Created using wingetcreate 1.5.3.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+# Created with komac v2.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
 
 PackageIdentifier: Poly.PlantronicsHub
 PackageVersion: 3.25.54065.37203
 InstallerLocale: en-US
-MinimumOSVersion: 10.0.0.0
-InstallerType: burn
+InstallerType: exe

```

Comment:
@russellbanks, I don't know why it changes it to exe instead of burn and removes "MinimumOS".